### ISSUE TITLE
Problem with variable products after update 2.6 to 3.1

### ISSUE THREAD

**Issue by poyagh** [woocommerce/woocommerce#16146]
Hi. I updated my woocommerce from 2.6 to 3.1 and then problems occurred and also in 3.1.1 not fixed.
1- When I go to a variable product editor my variations sections not load! It stay at loading!

![c3a4670a-3030-11e7-8229-a76b044c011c](https://user-images.githubusercontent.com/29813203/28274067-18db1532-6b25-11e7-8a3e-aad1d2256045.png)

2- Some products pages show a message that related to "This product is not accessible" and there is no add to card button. Where as when I go to product editor and see the Inventory section, I see that is In stock and no problem is there! But I should click on update button to fix this problem!
You can check this at:
http://bit.ly/2uvpn3v

3- When the user types an incorrect username or password and click on login button, the page will just refresh! Without any error message.
I disabled my Recaptcha plugin but it not resolved.
You can check this at:
http://bit.ly/2vuqkXk

**Comment by f3bruary** [comment]
Do you have WPML plugins enabled ?

**Comment by mikejolley** [comment]
1. https://github.com/woocommerce/woocommerce/wiki/Select2-fields-not-working-in-3.0.x

2. Means the combination of attributes matches a variation which is invalid

3. Will be a plugin/theme conflict.

2.6 -> 3.1 is still going to be a major update for you which requires you to check themes + plugins for compatibility. Read the post I linked to above and go from there.

Closing as this is not a valid bug report.
